projects/Mallard/src/helpers/weather-visibility.ts

```diff
@@ -0,0 +1,40 @@
+import { getSetting, storeSetting } from './settings'
```

Comment:
This should be in the hooks folder rather than helpers

Reply:
sure thing